oaknational/Oak-Web-Application
# feat: 1150 add filters to url

## Description

Added search filters to URL

## Issue(s)

Fixes #1150

## How to test

1. Go to search page and type a search term
2. Click on filters - they should be added to url
3. Go through pagination pages - they should be persisted
